src/components/LayerControl.vue

```diff
@@ -13,6 +13,9 @@
         @click="clicked(layer.id)"
       >{{ layer.name }}</div>
     </div>
+    <button class="clear-button" title="Clear layer" @click="clearLayer">
```

Comment:
This `title` attribute will need i18n support, but everything else looks good.

Reply:
good catch